.gitignore

```diff
@@ -1,4 +1,4 @@
-/target/
+target
 /target-tiny/
```

Comment:
I believe this is no longer needed. I realize this is out-of-scope of this PR, but with loose relevance we could remove too.
```suggestion
```